### PR TITLE
Add page for android apps

### PR DESCRIPTION
**Pull request by garethbowen**

### CODE REVIEW

**Review by mrsarm — APPROVED** [review]
LGTM.

File: content/en/apps/guides/hosting/android-app.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+- The app logo and title
+- Translations for your supported languages
+
+Follow these [instructions to create a new brand](https://github.com/medic/medic-android/#adding-new-brands).
+
+### Building APKs
+
+Use Fastlane to build and sign your APKs ready for publishing. Building can be done using CI (eg: GitHub Actions, travis ci). For reference, refer to [how Medic builds APKs using GitHub Actions](https://github.com/medic/medic-android/blob/master/.github/workflows/publish.yml).
+
+It's important to ensure you sign your APKs using the same certificate to ensure that upgrading to a new version goes smoothly.
```
Comment (kennsippell) [inline content/en/apps/guides/hosting/android-app.md:25]: maybe mention that the certificate is provided during the CI tool, and so this is why we don't recommend running fastlane locally to deply

Comment (garethbowen) [reply content/en/apps/guides/hosting/android-app.md]: Added.

**Comment by garethbowen** [comment]
I've merged this to get _something_ live. I'm happy to make further changes incrementally as needed.

**Pull request merged**